ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/Main.scala

```diff
@@ -87,6 +92,18 @@ object Main {
           val extra = config.extra
           config.copy(extra = extra.copy(maxInboundMessageSize = maxInboundMessageSize))
         })
+      parser
+        .opt[BatchingLedgerWriterConfig]("batching")
+        .optional()
+        .text(BatchingLedgerWriterConfigReader.UsageText)
+        .action({
+          case (parsedBatchingConfig, config) =>
+            config.copy(
+              extra = config.extra.copy(
+                batchingLedgerWriterConfig = parsedBatchingConfig
+              )
+            )
+        })
```

Comment:
The parentheses are unnecessary, I believe.

Reply:
Removed.